tools/templates/Makefile-common-rules.in

```diff
@@ -1,6 +1,6 @@
 # Copyright (C) 2015 The Perl Foundation
 
-.NOTPARALLEL:
+#.NOTPARALLEL:
```

Comment:
The reason for prohibiting parallel make was that I couldn't find a way to make it still reliably pick up changes in nqp. We really do need to run check-nqp-version.pl first. Is that problem solved?

Reply:
Never knew about this problem. Will see what could be done about it.

Reply:
https://github.com/rakudo/rakudo/commit/8ad7eb398a4a4ab04d4c109e93ff5d844eb308a1

Luckily, git remembers the whole story:

"Fix parallel initial build failing

tools/build/check-nqp-version.pl did run and produced a correct nqp-version
file, but at that point it was already too late. While it was running, make
tried to check the dependency for other parts and gen/nqp-version was not
available yet and there was no explicit recipe for creating it either.

Mark the Makefile as .NOTPARALLEL, which gets rid of the whole issue. This is
a GNU extension however, so the issue may still remain for other makes."